tracer/build/_build/Build.Steps.cs

```diff
@@ -31,14 +31,14 @@ partial class Build
     [Solution("Datadog.Trace.sln")] readonly Solution Solution;
     AbsolutePath MsBuildProject => RootDirectory / "Datadog.Trace.proj";
 
-    AbsolutePath OutputDirectory => RootDirectory / "bin";
+    AbsolutePath OutputDirectory => RootDirectory / "tracer" / "bin";
```

Comment:
nit: we should probably extract `RootDirectory / "tracer"` to a property. Not for this PR, just a note!